ts/Data/GoogleDataStore.ts

```diff
@@ -172,17 +192,27 @@ class GoogleDataStore extends DataStore {
             dataType: 'json',
             success: function (json: Highcharts.JSONType): void {
 
-                parseSheet(json);
+                fireEvent(
+                    store,
+                    'load',
+                    { json, enablePolling, dataRefreshRate },
+                    function (): void {
+                        store.parseSheet(json);
+                        const table = store.colsToDataTable();
 
-                // Polling
-                if (enablePolling) {
-                    setTimeout(
-                        function (): void {
-                            fetchSheet();
-                        },
-                        dataRefreshRate * 1000
-                    );
-                }
+                        // Polling
+                        if (enablePolling) {
+                            setTimeout(
+                                function (): void {
+                                    store.fetchSheet();
+                                },
+                                dataRefreshRate * 1000
+                            );
+                        }
+
+                        fireEvent(store, 'afterLoad', { table });
+                    }
+                );
```

Comment:
The event firing looks good, but I see no `on` function to attach a listener. Additionally the typing for the callback functions and the event properties are missing. See DataTable and DataRow for examples.